src/components/PeoplePicker/PeoplePicker.tsx

```diff
@@ -1,113 +1,215 @@
-import React from 'react';
-import { IPersonaProps } from '@fluentui/react/lib/Persona';
-import { IBasePickerSuggestionsProps, NormalPeoplePicker } from '@fluentui/react/lib/Pickers';
-import { people } from '@fluentui/example-data';
+import React from "react";
+import { IPersonaProps } from "@fluentui/react/lib/Persona";
+import {
+  IBasePicker,
+  IBasePickerSuggestionsProps,
+  NormalPeoplePicker,
+} from "@fluentui/react/lib/Pickers";
+import { people } from "@fluentui/example-data";
+import { spWebContext } from "../../providers/SPWebContext";
+import { IPeoplePickerEntity } from "@pnp/sp/profiles";
+
+// TODO: Add a way to show as input needed/corrected
 
 const suggestionProps: IBasePickerSuggestionsProps = {
-  suggestionsHeaderText: 'Suggested People',
-  mostRecentlyUsedHeaderText: 'Suggested Contacts',
-  noResultsFoundText: 'No results found',
-  loadingText: 'Loading',
+  suggestionsHeaderText: "Suggested People",
+  mostRecentlyUsedHeaderText: "Suggested Contacts",
+  noResultsFoundText: "No results found",
+  loadingText: "Loading",
   showRemoveButtons: true,
-  suggestionsAvailableAlertText: 'People Picker Suggestions available',
-  suggestionsContainerAriaLabel: 'Suggested contacts',
+  suggestionsAvailableAlertText: "People Picker Suggestions available",
+  suggestionsContainerAriaLabel: "Suggested contacts",
 };
 
-export const PeoplePicker: React.FunctionComponent<any> = (props) => {
-  const [currentSelectedItems, setCurrentSelectedItems] = React.useState<IPersonaProps[]>([]);
+export interface SPPersona extends IPersonaProps {
+  AccountName?: string;
+  Department?: string;
+  Email?: string;
+  SPUserId?: number;
+}
```

Comment:
Rather than rolling our own classes, should we utilize the existing pnp ones?
IPeoplePickerEntityData

Reply:
Agree that we need to standardize it-- I have a note that need to investigate how to tie the User API / People Picker / Avatar / SP Person field (PNP) together properly.    Turned it into #12 so we can track better than a "TODO"